docs/root/configuration/best_practices/_include/edge.yaml

```diff
@@ -61,30 +61,30 @@ static_resources:
           merge_slashes: true
           path_with_escaped_slashes_action: UNESCAPE_AND_REDIRECT
           common_http_protocol_options:
-            idle_timeout: 3600s  # 1 hour
+            idle_timeout: 3600s # 1 hour
             headers_with_underscores_action: REJECT_REQUEST
           http2_protocol_options:
             max_concurrent_streams: 100
-            initial_stream_window_size: 65536  # 64 KiB
-            initial_connection_window_size: 1048576  # 1 MiB
-          stream_idle_timeout: 300s  # 5 mins, must be disabled for long-lived and streaming requests
-          request_timeout: 300s  # 5 mins, must be disabled for long-lived and streaming requests
+            initial_stream_window_size: 65536 # 64 KiB
+            initial_connection_window_size: 1048576 # 1 MiB
+          stream_idle_timeout: 300s # 5 mins, must be disabled for long-lived and streaming requests
+          request_timeout: 300s # 5 mins, must be disabled for long-lived and streaming requests
           http_filters:
           - name: envoy.filters.http.router
             typed_config:
               "@type": type.googleapis.com/envoy.extensions.filters.http.router.v3.Router
           route_config:
             virtual_hosts:
             - name: default
-              domains: ["*"]
+              domains: [ "*" ]
```

Comment:
this is the exception where not using proper yaml is generally allowed - but no adding in annoying ws is defo not an improvement